### ISSUE TITLE
CocoNetError: ValueError: Can't load save_path when it is None.

### ISSUE THREAD

**Issue by SYaddanapudi** [tensorflow/magenta#1587]
I am trying to run a pretrained model and I keep on getting this error. Below is the command I am running
 sh sample_bazel.sh `/Users/Jason/Documents/GitHub/magenta/magenta/models/coconet/coconet-64layers-128filters `

Error Message:

```
File "/Users/Jason/.conda/envs/tensorflow1.6.0/lib/python3.6/site-packages/tensorflow/python/training/saver.py", line 1274, in restore
     raise ValueError("Can't load save_path when it is None.")
 ValueError: Can't load save_path when it is None.
```

**Comment by sathishkumarnarayanasamy** [comment]
Please replace the file "checkpoint" in your coconet_checkpoint\coconet-64layers-128filters folder with the following one: 
[checkpoint.zip](https://github.com/tensorflow/magenta/files/3763823/checkpoint.zip).

**Comment by jonnybluesman** [comment]
> Please replace the file "checkpoint" in your coconet_checkpoint\coconet-64layers-128filters folder with the following one: [checkpoint.zip](https://github.com/tensorflow/magenta/files/3763823/checkpoint.zip).
> This is not an issue in code. we can close this item.

This is not correct. To actually fix the problem, you need to update the `checkpoint` file as discussed in a [previous issue](https://github.com/magenta/magenta/issues/1292).
